.github/workflows/tests.yml

```
@@ -9,7 +9,7 @@ jobs:
         runs-on: ubuntu-latest
         strategy:
             matrix:
-                php: [ 5.6, "7.0", 7.1, 7.2, 7.3 ]
+                php: [ 5.6, 7.0, 7.1, 7.2, 7.3, 7.4, 8.0, 8.1 ]
```

nit, but the quotes are needed around `#.0` releases, otherwise the names display as the number without the decimal.
```suggestion
                php: [ 5.6, "7.0", 7.1, 7.2, 7.3, 7.4, "8.0", 8.1 ]
```